Update Exchange-Teams-interact.md (#7918)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/Exchange-Teams-interact.md

<sup>9</sup> A minimum of an Exchange Online Plan 1 license is also required. For more information, see [Search for Teams chat data for on-premises users](/microsoft-365/compliance/search-cloud-based-mailboxes-for-on-premises-users).

<sup>10</sup> On-premises users can use Teams to update their profile picture even if the `SetPhotoEnabled` Outlook on the Web mailbox policy is set to `false`.
 > [!NOTE]

>  > Setting OOF via the teams client is currently not supported for users that their mailbox is hosted onprem, users should perform this action via the outlook client.

Reviewer: ```suggestion
 > Setting Out of Office via the Teams client is currently not supported for users whose mailboxes are hosted on-premises; these users should perform this action via the Outlook client.
```
